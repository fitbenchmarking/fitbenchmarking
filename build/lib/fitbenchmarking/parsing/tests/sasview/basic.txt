# Fitbenchmark Problem
software = 'Sasview'
name = 'basic'
description = 'This dataset has no real world application. It is used to test the parsing.'
input_file = 'sasview_basic.dat'
function ='name=cylinder,radius=0.0,length=1.0'