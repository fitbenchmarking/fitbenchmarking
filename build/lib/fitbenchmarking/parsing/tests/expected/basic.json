{"name": "basic",
 "start_x": null,
 "end_x": null,
 "data_x": [0.5, 1.0, 1.5, 2.0, 2.5],
 "data_y": [33.5, 1.0, 0.1, -2.0, 0],
 "data_e": null,
 "function": "Single_function_2_params",
 "starting_values": [{"p1": 0.0, "p2": 1.0}],
 "value_ranges": "[(0.0, 10.0), (0.0, 10.0)]"
}